blocks/layout-grid/src/grid/resize-grid/index.js

```diff
@@ -170,28 +192,46 @@ class ResizeGrid extends Component {
 		if ( adjustment ) {
 			this.props.onResize( this.state.resizingColumn, adjustment );
 		}
-	}
+	};
 
-	onMouseUp = ev => {
-		this.setState( { resizingColumn: -1 } );
+	onMouseUp = ( ev ) => {
+		if ( this.state.resizingColumn === -1 ) {
+			return;
+		}
 
+		this.setState( { resizingColumn: -1 } );
 		document.removeEventListener( 'mousemove', this.onMouseMove );
-		document.removeEventListener( 'mouseup', this.onMouseUp );
```

Comment:
This and line 181 looks like the only change that isn't formatting.

Reply:
Yeah, that's correct. The first commit on this branch is just linting. It's going to be a thing for the whole package, as the npm versions were really out of date.